pom.xml

```diff
@@ -74,7 +74,7 @@ limitations under the License.
     <guava.version>18.0</guava.version>
     <joda.version>2.8.1</joda.version>
     <h2.version>1.4.185</h2.version>
-    <hadoop.version>2.6.0</hadoop.version>
+    <hadoop.version>2.7.0</hadoop.version>
```

Comment:
Had to upgrade to 2.7 to work around a broken Guava dependency in 2.6. https://issues.apache.org/jira/browse/HADOOP-10101